core/src/main/java/org/hibernate/ogm/exception/spi/ErrorHandler.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.exception.spi;
```

Comment:
@emmanuelbernard, given the definition of "SPI" we use, should this actually not rather be "API"? It's expected to be implemented and used by OGM users, not dialect authors or integrators. I think we should drop the "spi" from the package names.